src/main/scala/dpla/ingestion3/OaiHarvesterMain.scala

```diff
@@ -80,38 +105,13 @@ object OaiRecordHarvesterMain {
       .option("avroSchema", schemaStr)
       .avro(oaiConf.outputDir())
 
+    println(dataframe.show) // For testing only, can delete later.
```

Comment:
I'll just delete it, I don't think it's worth logging.  But good suggestion for future debugging!